pkg/network/tracer/testutil/cmd/gotls_client/gotls_client.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		log.Fatalf("usage: %s <server_addr> <number_of_requests>", os.Args[0])
+	}
+
+	serverAddr := os.Args[1]
+	reqCount, err := strconv.Atoi(os.Args[2])
+	if err != nil || reqCount < 0 {
+		log.Fatalf("invalid value %q for number of request", os.Args[2])
+	}
+
+	client := http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+
+	// Needed to give time to the tracer to hook GoTLS functions
+	time.Sleep(1 * time.Second)
```

Comment:
didn't we want to split it for "setup" and "run" to avoid races?

Reply:
Ideally that's what we would want to do. However, we currently have no way at the moment to know when the probes are actually attached on the process.